### DISCUSSION TITLE
Welcome to am-editor Discussions!

### DISCUSSION THREAD

**Discussion by big-camel** [red-axe/am-editor#47] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by blog-xylia** [comment]
原内容如下，多序列列表
![541690278488_ pic](https://github.com/red-axe/am-editor/assets/40161722/1d301a61-2a0b-40e9-bd15-83c66efc4f70)
粘贴一段有格式的内容，多序列列表序号乱串。
![551690278490_ pic](https://github.com/red-axe/am-editor/assets/40161722/15b8bf68-c34a-435d-bed1-c9a8fe63e4be)
请问如何解决？

**Comment by hasnum-stack** [comment]
hi 目前咱们仓库还维护么

**Reply by big-camel** [reply]
> hi 目前咱们仓库还维护么

看起来是没有人维护了
